lua/rust-tools/dap.lua

```diff
@@ -1,30 +1,25 @@
+local config = require('rust-tools.config')
 local M = {}
 
 function M.setup_adapter()
     local dap = require('dap')
-    dap.adapters.rt_lldb = {
+    dap.adapters.lldb = {
         type = 'executable',
         command = 'lldb-vscode',
-        name = "rt_lldb"
+        name = "lldb"
     }
 end
 
 local function get_cargo_args_from_runnables_args(runnable_args)
     local cargo_args = runnable_args.cargoArgs
 
+    table.insert(cargo_args, '--no-run')
     table.insert(cargo_args, '--message-format=json')
 
     for _, value in ipairs(runnable_args.cargoExtraArgs) do
         table.insert(cargo_args, value)
     end
 
-    if not vim.tbl_isempty(runnable_args.executableArgs) then
```

Comment:
Similarly, there's no need to pass extra arguments if we don't run the binary directly.